q2_qemistree/citations.bib

```diff
@@ -0,0 +1,14 @@
+@article{letunic2019itol,
+    author = {Letunic, Ivica and Bork, Peer},
+    title = "{Interactive Tree Of Life (iTOL) v4: recent updates and new developments}",
+    journal = {Nucleic Acids Research},
+    volume = {47},
+    number = {W1},
+    pages = {W256-W259},
+    year = {2019},
+    month = {04},
+    issn = {0305-1048},
+    doi = {10.1093/nar/gkz239},
+    url = {https://doi.org/10.1093/nar/gkz239},
+    eprint = {https://academic.oup.com/nar/article-pdf/47/W1/W256/28879775/gkz239.pdf},
+}
```

Comment:
Thanks a lot for adding these. Can we add a citation for Sirius+CSI:FingerID as well? (PMID: 26392543)

Reply:
Good call, I just added classyfire as well.